### DISCUSSION TITLE
Release 1.10.1

### DISCUSSION THREAD

**Discussion by dreua** [pdfarranger/pdfarranger#992] [Announcements]
Fedora 39 will be released with Python 3.12 in the next weeks, other distros might switch to it as well.
I'm wondering if we should make a new release or bugfix release in order to minimize impact by #990 .

Otherwise I'd need to patch at least the Fedora packaging.

**Comment by jeromerobert** [comment]
I was thinking of doing a 1.10.1 with only #987, #990, 3477ee615d089483d36337a9c1adeda4d97b3df3, 438e7836353ca263c7f9f9d8bcb0a893b5e9b765. Also may be some translation update.

**Comment by jeromerobert** [comment]
Well I'm not sure 1.10 is compatible with PikePDF 8 so it probably make no sense to do a 1.10.1. But I also think it's not reasonable to do a release before december. We have ongoing changes in the Windows build which will needs time to be tested.

**Reply by dreua** [reply]
What exactly might be incompatible? 1.10.0 runs fine with the latest pikepdf, I update it in the Flatpak all the time.

**Reply by jeromerobert** [reply]
Ok, thanks. Then I'll do a 1.10.1 with minimal changes ASAP.

**Reply by jeromerobert** [reply]
Done

**Reply by dreua** [reply]
Thank you!
